modules/pages/import/config.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+const Import = () => import('./pages/import/index').then(m => m.default || m);
+const privileges = {
+    IMPORT: ['IMPORT_READ'],
+};
+
+export default {
+    router: [
+        {
+            name: 'import',
+            path: '/import',
+            component: Import,
+            meta: {
+                privileges: privileges.IMPORT,
+            },
+            // children: [
```

Comment:
There isn't any todo comment, should it exist in code?

Reply:
this is example to know how its work.

Reply:
Add proper comment then :+1: